pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -42,8 +42,7 @@ class EarlyStopping(Callback):
     r"""
 
     Args:
-        monitor: quantity to be monitored. Default: ``'val_loss'``.
-            .. note:: Has no effect when using `EvalResult` or `TrainResult`
+        monitor: quantity to be monitored. Default: ``'early_stop_on'``.
```

Comment:
can the early stopping changes be split out to a separate PR?

Reply:
yeah, i initially tried but since val_loss was used for both it made it super difficult